spec/mysql2/statement_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe Mysql2::Statement do
   before :each do
+    @client.close
```

Comment:
Feels odd to close the client before instantiating it. Should this be in an `after { … }`?